dev apps/XForms/XForms.iOS/AppDelegate.cs

```diff
@@ -57,7 +57,11 @@ public override bool FinishedLaunching(UIApplication app, NSDictionary options)
 
         public override bool OpenUrl(UIApplication app, NSUrl url, NSDictionary options)
         {
-            AuthenticationContinuationHelper.SetAuthenticationContinuationEventArgs(url, "");
+            if (!AuthenticationContinuationHelper.SetAuthenticationContinuationEventArgs(url, ""))
+            {
+                return false;
```

Comment:
No I cannot because there could be other apps that may be invoking this app to send data. Example Uber can be invoked by FB messenger, browser etc. each invoker could be sending different data to the app.